methods/push.js

```diff
@@ -0,0 +1,10 @@
+const myLength = require('./length');
+
+function myPush(array, value) {
+  const length = myLength(array);
+  const newArr = array;
```

Comment:
this is dangerous, what happens here is that you set a pointer to an array named `array` which is the original array. You can potentially mutate the original array like this. I suggest to **always** work with copy of the the original array